### PR TITLE
php-function-strtolower

### LINKED ISSUE #2435: [Term Entry] PHP String Functions strtolower()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `strtolower()` function.

The entry should go in a new file in `docs/content/php/concepts/string-functions/terms/strtolower/strtolower.md`

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

Thanks for assigning me. This closes #2435 .

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/strtolower/strtolower.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`strtolower()`** method in PHP converts a string to lowercase. It takes a string as input and returns a new string with all the characters converted to lowercase.
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/strtolower/strtolower.md:16]:
```suggestion
The **`strtolower()`** function in PHP converts a string to lowercase. It takes a string as input and returns a new string with all the characters converted to lowercase.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@nikolaspoczekaj Looks good, only one comment.

**Comment by nikolaspoczekaj** [comment]
@SSwiniarski Done with the changes !

**Review by caupolicandiaz — APPROVED** [review]
@nikolaspoczekaj @SSwiniarski LGTM

**Comment by SSwiniarski** [comment]
@nikolaspoczekaj Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/string-functions/strtolower

image

**Pull request merged**